tests/cases/fourslash/trailingCommaSignatureHelp.ts

```diff
@@ -7,12 +7,24 @@
 //// */
 ////function str(n: number, radix: number): string;
 ////function str(n: number, radix?: number): string { return ""; }
+////
+////str(1, /*a*/)
+////
+////declare function f<T>(a: T): T;
+////f(2, /*b*/);
 
-edit.insert("str(1,");
-verify.signatureHelp({
-    text: "str(n: number, radix: number): string",
-    parameterName: "radix",
-    parameterDocComment: "The radix",
-    docComment: "Stringifies a number with radix",
-    tags: [{ name: "param", text: "radix The radix" }],
-});
+verify.signatureHelp(
+    {

```

Comment:
Previously this test was wrong -- the `edit.insert` call applied to the start of the file, changing the first line to `str(2,|function str(n: number): string;` and eliminating that overload. The correct behavior still *has* two overloads, but shows the one with higher arity first.

As for the second test showing `{}` instead of `number`, that's because in signature help, we assume a trailing comma indicates trying to write another argument, which means `chooseOverload` fails and we go to `getCandidateForOverloadFailure` which doesn't infer type arguments.